io.catenax.essincident/2.0.0/EssIncident.ttl

```diff
@@ -0,0 +1,506 @@
+
+:productCommodity a samm:Property ;
+   samm:preferredName "Product Commodity"@en ;
+   samm:description "Free-text description for commodity of a product affected by an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Tire" .
+
+:productDescription a samm:Property ;
+   samm:preferredName "Product Description"@en ;
+   samm:description "Description of product or component affected by an incident in the context of ESS (Environmental and Social Standards) "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Natural Rubber" .
+
+:partNumber a samm:Property ;
+   samm:preferredName "Part Number"@en ;
+   samm:description "Part number that is affected by an incident in the context of ESS (Environmental and Social Standards) "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "EX123M234" .
+
+:rawMaterial a samm:Property ;
+   samm:preferredName "Raw Material"@en ;
+   samm:description "Raw material that causes an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Natural Rubber" .
+
+:industry a samm:Property ;
+   samm:preferredName "Industry / Branch"@en ;
+   samm:description "Industry / Branch that causes the incident"@en ;
+   samm:characteristic :IndustryCharacteristic ;
+   samm:exampleValue "Extraction of raw material" .
+
+:essOriginatorCountry a samm:Property ;
+   samm:preferredName "ESS Originator Country"@en ;
+   samm:description "Country to which an incident in the context of ESS (Environmental and Social Standards) belongs "@en ;
+   samm:characteristic :EssOriginatorCountryTrait ;
+   samm:exampleValue "BR" .
+
+:essOriginatorCountrySubdivision a samm:Property ;
+   samm:preferredName "ESS Originator Country Subdivision"@en ;
+   samm:description "Region within a country to which an incident in the context of ESS (Environmental and Social Standards) belongs"@en ;
+   samm:characteristic :EssOriginatorRegionTrait ;
+   samm:exampleValue "BR-AM" .
+
+:essOriginatorCoordinates a samm:Property ;
+   samm:preferredName "ESS Originator Geographic data / Coordinates"@en ;
+   samm:description "Exact geographic position of an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic :EssOriginatorCoordinatesCharacteristic .
+
+:essOriginatorCompanyName a samm:Property ;
+   samm:preferredName "ESS Originator Company Name"@en ;
+   samm:description "Name of a company / an organisation that is the originator of an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Rubbery Ltd." .
+
+:address a samm:Property ;
+   samm:preferredName "Address"@en ;
+   samm:description "Simple form of an address which can belong to a person, organisation, company etc."@en ;
+   samm:characteristic ext-characteristic:PostalAddress .
```

Comment:
this reference statement is giving error. Reason: the referenced object is still written in the 'bamm' and not 'samm'. hence this model is giving error in validation.
Suggestion would be to create a new referenced model (io.catenax.shared.address_characteristic:2.0.0) with 3.0.0 version which is written in 'samm'.